commons/commons-model-extended/src/main/kotlin/ebi/ac/uk/extended/model/ExtendedModel.kt

```diff
@@ -107,19 +107,29 @@ data class ExtSection(
 
 data class ExtAccessTag(val name: String)
 
+interface ExtSubmissionInfo {
```

Comment:
created this one to avoid requiring to load all submission only for a few fields.

Reply:
fantastic idea yes, doing this